### ISSUE TITLE
v2.28.0 not working on Windows 7

### ISSUE THREAD

**Issue by cplerch** [alm-tools/alm-tools.github.io#4]
Installed alm.tools on Windows 7 x64 using npm v3.10.9  on node v6.9.1
When starting `alm` (or `alm -i`) I'm getting:
```
F:\dev\trial\alm\test-proj1>alm
Version: 2.28.0
DASHBOARD: (Please open in chrome) http://localhost:4444
F:\data\user.cpl\AppData\Roaming\npm\node_modules\alm\src\server\workers\lang\core\simpleValidator.js:34
                        if (!validValues.some(function (valid) { return valid.toLowerCase() === value_1.toLowerCase(); })) {
                                                                                                        ^
TypeError: value_1.toLowerCase is not a function
```

**Comment by basarat** [comment]
Can you share you `tsconfig.json` when that error occurs :rose:

**Comment by cplerch** [comment]
Sure, that's what has been auto-generated by `alm -i`:
```
{
  "compilerOptions": {
    "target": "es5",
    "module": "commonjs",
    "moduleResolution": "node",
    "jsx": 0,
    "experimentalDecorators": false,
    "emitDecoratorMetadata": false,
    "declaration": false,
    "noImplicitAny": false,
    "suppressImplicitAnyIndexErrors": false,
    "strictNullChecks": false
  },
  "compileOnSave": true,
  "exclude": [
    "node_modules",
    "typings/browser",
    "typings/browser.d.ts"
  ]
}
```
It's an empty new project, didn't change or add anything. Then after trying to start `alm` got the error as indicated before and the server was not available at `http://localhost:4444`.

**Comment by paganaye** [comment]
Reporting exact same issue on Linux Mint

**Comment by basarat** [comment]
got it. Basically `alm -i`  is broken as it generates

> "jsx": 0,

The validator expects a string value but it comes out as a number. Working on fixing 🌹

* [x] alm -i should not generate a bad config file 
* [x] simple validator should check `typeof` before doing `value.toLowerCase`

**Comment by basarat** [comment]
Fixed with `2.29.0` :rose: Thanks for your patience :heart:
